presentation/matrix_view.py

```diff
@@ -203,53 +208,71 @@ def get_thresholds(self, metric: MetricType) -> Thresholds:
         return self._config.packet_loss
 
     @staticmethod
-    def get_metric_value(metric: MetricType, cell: MeshColumn) -> MetricValue:
+    def get_metric_value(metric: MetricType, health: HealthItem) -> MetricValue:
         if metric == MetricType.LATENCY:
-            return cell.latency_millisec.value
+            return health.latency_millisec
         if metric == MetricType.JITTER:
-            return cell.jitter_millisec.value
-        return cell.packet_loss_percent.value
+            return health.jitter_millisec
+        return health.packet_loss_percent
```

Comment:
We need to have `if` branch for every entry in the `MetricType` enum and raise an exception by default (if an unknown value is passed in). That is more future proof and guards against programming errors (incomplete changes).